device_backends/include/VariableDistributor.h

```diff
@@ -0,0 +1,119 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#pragma once
+
+#include "AsyncAccessorManager.h"
+#include "InterruptControllerHandler.h"
+
+#include <memory>
+
+namespace ChimeraTK {
+
+  template<typename SourceType>
+  class VariableDistributor : public AsyncAccessorManager {
+   public:
+    VariableDistributor(boost::shared_ptr<DeviceBackend> backend, std::vector<uint32_t> interruptID,
+        boost::shared_ptr<TriggerDistributor> parent, boost::shared_ptr<AsyncDomain> asyncDomain);
+
+    template<typename UserType>
+    std::unique_ptr<AsyncVariable> createAsyncVariable(AccessorInstanceDescriptor const& descriptor);
+
+    void activate(SourceType, VersionNumber version);
+
+    void distribute(SourceType, VersionNumber version);
+
+    std::vector<uint32_t> _id;
+    boost::shared_ptr<TriggerDistributor> _parent;
+    SourceType _sourceBuffer;
+    VersionNumber _version{nullptr};
+  };
+
+  //*********************************************************************************************************************/
```

Comment:
Hm, strange. I though I checked that they all end on character 119.

Reply:
That was probably before changing the indentation for some reason :-)

Reply:
Oh dear. There are hundreds of too long chrome bars all over the place, also in files I never touched.

Reply:
572 crome bars are too long... I will systematically replace them.

Reply:
I will make a separate ticket and do it later. It has to be a separate commit because it touches half of the files all over DeviceAccess to do it systematically. If I put that in now, I will get merge conflicts if  try to squash my changes later.

Reply:
https://redmine.msktools.desy.de/issues/12914